<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>getuserdialogClass</class>
 <widget class="QDialog" name="getuserdialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>366</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>312</width>
    <height>366</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>312</width>
    <height>366</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="checkBox_IdUser">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Identifiant</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Start">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>131</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Lancer recherche</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_lastName">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_lastName">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_IdUser">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_firstName">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Pr√©nom</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_firstName">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
